.github/docker-entrypoint.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+set -e
+
+REPLICA_SET_NAME=${REPLICA_SET_NAME:=rs0}
+USERNAME=${USERNAME:=dev}
+PASSWORD=${PASSWORD:=dev}
+
+
+function waitForMongo {
+    port=$1
+    n=0
+    until [ $n -ge 20 ]
+    do
+        mongo admin --quiet --port $port --eval "db" && break
+        n=$[$n+1]
+        sleep 2
+    done
+}
+
+if [ ! "$(ls -A /data/db1)" ]; then
+    mkdir /data/db1
+    mkdir /data/db2
+    mkdir /data/db3
+fi
+
+echo "STARTING CLUSTER"
+
+mongod --port 27003 --dbpath /data/db3 --replSet $REPLICA_SET_NAME --bind_ip=::,0.0.0.0 &
+DB3_PID=$!
+mongod --port 27002 --dbpath /data/db2 --replSet $REPLICA_SET_NAME --bind_ip=::,0.0.0.0 &
+DB2_PID=$!
+mongod --port 27001 --dbpath /data/db1 --replSet $REPLICA_SET_NAME --bind_ip=::,0.0.0.0 &
+DB1_PID=$!
+
+waitForMongo 27001
+waitForMongo 27002
+waitForMongo 27003
+
+echo "CONFIGURING REPLICA SET"
+CONFIG="{ _id: '$REPLICA_SET_NAME', members: [{_id: 0, host: 'localhost:27001', priority: 2 }, { _id: 1, host: 'localhost:27002' }, { _id: 2, host: 'localhost:27003' } ]}"
+mongo admin --port 27001 --eval "db.runCommand({ replSetInitiate: $CONFIG })"
+
+waitForMongo 27002
```

Comment:
While this might be sufficient I think it's operationally different. [rs_status](https://github.com/node-casbin/mongoose-adapter/pull/28/files#diff-de4af981e1065047e2fdb5c41a1b8be12903f6762a25e06bbe3db76d8a5bfb98) previously tried to verify that replicaset is valid.

The difference here might be that `db`-command could go through even when replicaset configuration is not yet done. (For example voting for primary / secondary / arbiter might not be done yet, but database accepts requests)

While `waitForMongo` can be sufficient to get the database responsive, I am not sure whether it means that the replicaset is fully operational.

This might be just splitting hairs , but I would try to avoid any unexpected errors.

Reply:
I think this won't be a problem. Since after this CI will execute `yarn install` then.
Mongo has enought time to finish its work.

Reply:
Yeah, you are probably right.